interface/modules/custom_modules/oe-module-prior-authorizations/public/reports/list_report.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ *  package OpenEMR
+ *  link    https://www.open-emr.org
+ *  author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  Copyright (c) 2022.
+ *  All Rights Reserved
+ */
+
+use OpenEMR\Core\Header;
+use Juggernaut\OpenEMR\Modules\PriorAuthModule\Controller\AuthorizationService;
+
+require_once dirname(__FILE__, 6) . "/globals.php";
+require_once dirname(__FILE__, 3) . "/vendor/autoload.php";
+
```

Comment:
assuming need some sort of acl access here

Reply:
Again, none requested